tasks/tomcat_wars.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: "Ensure webapp dir exists (url)"
+  shell: "mkdir -p {{ item.directory | default(tomcat_directory) }}/{{ tomcat_archive[item.version | default (tomcat_version)]['dir'] }}/webapps"
+  when:
+    - "tomcat_wars.url is defined"
+  with_items: "{{ tomcat_wars.url }}"
+- name: "loop over tomcat_wars"
```

Comment:
Empty lines between tasks would improve readability.

Reply:
Yeah, will fix. Had issues with molecule until I learned to use it.